Source/Core/HermiteSpline.js

```diff
@@ -122,14 +122,14 @@ function generateNatural(points) {
  * @param {Object} options Object with the following properties:
  * @param {Number[]} options.times An array of strictly increasing, unit-less, floating-point times at each point.
  *                The values are in no way connected to the clock time. They are the parameterization for the curve.
- * @param {Cartesian3[]} options.points The array of {@link Cartesian3} control points.
- * @param {Cartesian3[]} options.inTangents The array of {@link Cartesian3} incoming tangents at each control point.
- * @param {Cartesian3[]} options.outTangents The array of {@link Cartesian3} outgoing tangents at each control point.
+ * @param {Number[]|Cartesian3[]|Quaternion[]} options.points The array of control points.
+ * @param {Number[]|Cartesian3[]|Quaternion[]} options.inTangents The array of incoming tangents at each control point.
+ * @param {Number[]|Cartesian3[]|Quaternion[]} options.outTangents The array of outgoing tangents at each control point.
```

Comment:
In another comment you mentioned that quaternions will be implemented later, should we remove them from the JSDoc for now if they're not supported?

Reply:
Okay, sure. I'm removing `Number` and `Quaternion`s from the documentation until they are properly supported.